src/log_surgeon/Token.cpp

```
@@ -1,6 +1,8 @@
 #include "Token.hpp"
 
 #include <cstddef>
+#include <optional>
```

Cppcheck (2.18.0)

[information] Include file

(missingIncludeSystem)